awx/main/utils/common.py

```diff
@@ -435,16 +440,50 @@ def model_to_dict(obj, serializer_mapping=None):
     return attr_d
 
 
+class CharPromptDescriptor:
+    """Class used for identifying nullable launch config fields from class
+    ex. Schedule.limit
+    """
+    def __init__(self, field):
+        self.field = field
+
+
+class NullablePromptPsuedoField:
```

Comment:
Can we fix the name here?

Reply:
Oh, you just mean `Psuedo` -> `Pseudo`

Yeah, of course. Thanks for finding all of these :-P